requirements.txt

```diff
@@ -430,9 +433,9 @@ mccabe==0.6.1 \
     # via
     #   -r requirements.in
     #   flake8
-mozanalysis==2021.1.2 \
-    --hash=sha256:af8d5d612a41d821b4bb8a17a07590288c6e2bb2cc2fd8c1873c66f60dc955d5 \
-    --hash=sha256:ce6b5d7da287407e43ed909dd7dc20acd4c6f4111caa58a5762aed0c888614c0
+mozanalysis==2021.2.1 \
+    --hash=sha256:22279da5fdeeeec32501ff0bce50e4c60d2d078211ae4ac95f16ee4e73711b11 \
+    --hash=sha256:5e47dc08b9d5f759e8abe82074758bd11f41f7a26449fd0a910dba7b28e26e7a
```

Comment:
This shouldn't break anything, I think. But the enrollments table schema will be different. I don't think we directly access them from anywhere though.

Reply:
Looks like integration tests are failing. Let's wait with upgrading mozanalysis until https://github.com/mozilla/jetstream/pull/509 gets merched